src/dumpContent.ts

```diff
@@ -0,0 +1,112 @@
+#!npx vite-node
+import fs from 'fs'
+import { promisify } from 'util'
+
+type ObjData = { _id: string }
+type SearchData = { continuation?: string; objs: ObjData[] }
+type BlobsData = { private_access: { get: { url: string } } }
+
+const DIRECTORY = 'contentDump'
```

Comment:
First I would place this in `src/` since it will (most likely with the next PR) be used by the import script that is included in the SPA.

Reply:
The import script is happy with this path. My local prototype can already restore from there.
Would you make it part of `src` nevertheless?

Reply:
No, thats ok. Let's leave it in `contentDump` then.